cdap-ui/app/directives/log-viewer/log-viewer.html

```diff
@@ -184,10 +179,6 @@
                 </span>
             </td>
 
-            <td class="source"
```

Comment:
do any of the stylings have to be updated to account for one less column?

Reply:
No. It was not needed. One thing good about logviewer 😑 . Since the columns are collapsible removing one column didn't need any styling changes.